openedx/features/calendar/models.py

```diff
@@ -0,0 +1,20 @@
+"""
+Models for Calendar
+"""
+
+
+from django.contrib.auth.models import User
+from django.db import models
+
+from opaque_keys.edx.django.models import CourseKeyField
+
+
+class UserCalendar(models.Model):
```

Comment:
Should this be named UserCalendarConfig or something? Rather than it being the object itself?

Or even, UserCalendarSyncConfig, because this is toggling the sync/email functionality. Or if not that, maybe rename `enabled` to `sync_enabled` or something.